### ISSUE TITLE
Add an --outdir option to Lore

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#11557]
|<img alt="spiv's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| spiv reported|
|-|-|
|Trac ID|trac#112|
|Type|enhancement|
|Created|2003-07-21 06:49:57Z|

<details><summary>Searchable metadata</summary>

```
trac-id__112 112
type__enhancement enhancement
reporter__spiv spiv
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__wontfix wontfix
component__lore lore
keywords__EndOfLore EndOfLore
time__1058770197000000 1058770197000000
changetime__1465030153780495 1465030153780495
version__ 
owner__ 
cc__spiv
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="spiv's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|spiv commented|
|-|-|

```
#!html
<pre>
Jeffrey Elkner asks:
> 1. Can I have lore read the source file from one directory
and put the
> target file in another?  I'm looking for something like:
> 
> lore --config template="template.tpl"> --indir source/rtn
 --outdir
> target/rtn
> 

Currently, there's no way to do this.  I don't think an
--indir option is needed thanks to shell globbing, but a
--outdir would probably be worthwhile.

</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun commented|
|-|-|

I think this would be cool.

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_3"></a>|@glyph set owner to @khorn|
|-|-|

Hopefully you can just close this as wontfix when you're done with the migration, but I'm going to start piling Lore bugs on you to remind you of the maintenance that you're supposedly going to save us from :).

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_4"></a>|@khorn commented|
|-|-|

I'll take this as tacit authorization to "wontfix" all these old Lore tix, in which case...

Bring it!

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_5"></a>|@glyph commented|
|-|-|

Replying to [khorn](#note_4):
> I'll take this as tacit authorization to "wontfix" all these old Lore tix, in which case...
> 
> Bring it!

You only get to do that _once the new system is in place_ :).

But yes, if we switch over to another tool, we can stop maintaining lore.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_7"></a>|@khorn commented|
|-|-|

Replying to [glyph](#note_5):
> Replying to [khorn](#note_4):
> > I'll take this as tacit authorization to "wontfix" all these old Lore tix, in which case...
> > 
> > Bring it!
> 
> You only get to do that _once the new system is in place_ :).
> 
> But yes, if we switch over to another tool, we can stop maintaining lore.


It's time. :D

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_8"></a>|@khorn set status to `closed`|
|-|-|

Lore is no longer being used for Twisted documentation, and is being deprecated, prior to eventual removal, per [#6907](https://github.com/twisted/twisted/issues/6907).

This issue will not be fixed.

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_9"></a>|@cyli commented|
|-|-|

In changeset ee98a62f8d0196c9b6fddf7d91a5724c7cc47725
```
#!CommitTicketReference repository="" revision="ee98a62f8d0196c9b6fddf7d91a5724c7cc47725"
Merge pull request #112 from rodrigc/8396-rodrigc-print: Convert from print statement to print() function in twisted/web

Author: rodrigc
Reviewer: cyli
Fixes: #8396

Convert from print statement to print() function in twisted/web
```
